check.ts

```diff
@@ -0,0 +1,140 @@
+import { Request, Response } from 'express';
+
+import ReviewService from '@/services/reviewService';
+import type { PullRequestWebhookEvent, PullRequestReviewWebhookEvent, IssueCommentWebhookEvent } from '@/types';
+import config from '@/utils/config';
+import logger from '@/utils/logger';
+
+class WebhookController {
+  private reviewService: ReviewService;
+
+  constructor() {
+    this.reviewService = new ReviewService();
+  }
+
+  async handlePullRequest(req: Request<{}, {}, PullRequestWebhookEvent>, res: Response): Promise<void> {
+    try {
+      const { action, number, pull_request, repository } = req.body;
+
+      logger.info('Received PR webhook', {
+        action,
+        number,
+        repo: repository?.full_name,
+      });
+
+      // Only process certain actions
+      const reviewActions = ['opened', 'synchronize', 'reopened'];
+      if (!reviewActions.includes(action)) {
+        logger.info('Ignoring PR action', { action });
+        return res.status(200).json({ message: 'Action ignored' });
+      }
+
+      // Only process PRs for configured repository
+      // const owner = repository.owner.login;
```

Comment:
Consider removing this commented code block or adding a clear TODO comment explaining why it's temporarily disabled. Dead code reduces readability and can cause confusion for other developers. ```suggestion
      // TODO: Add repository validation once configuration is finalized
      // Reference: https://github.com/your-org/repo/issues/123
```